compute_endpoint/globus_compute_endpoint/self_diagnostic.py

```diff
@@ -162,7 +162,6 @@ def run_self_diagnostic(log_bytes: int = 0):
         "ip route",
         "netstat -r",
         get_executable_path("globus-compute-endpoint"),
-        get_executable_path("process_worker_pool.py"),
         "globus-compute-endpoint whoami",
```

Comment:
Why remove the process_worker_pool.py exe path diagnostic?  Isn't it still useful for people who don't use the new python-exec command and run into issues?

I mean, we'll add it to the startup script path directly soon so prob not that important.

Reply:
I suppose it won't hurt to leave it in, but anyone using a version of the CLI that includes this change will not need need to resolve `process_worker_pool.py`. Thus, I don't see a need to include it in the diagnostic output.